content/tutorials/reactor-event-ifttt.textile

```diff
@@ -0,0 +1,196 @@
+
+
+<a href="/images/tutorials/reactor/reactor-event-ifttt-game.png" target="_blank">
+  <img src="/images/tutorials/reactor/reactor-event-ifttt-game.png" style="width: 100%" alt="screenshot of the game we will be making">
+</a>
+
+h2. Prerequisite Accounts
+
+You'll need an Ably account, login or sign up for free at "https://www.ably.io/":https://www.ably.io/.
+You'll also need an IFTTT account, login or sign up at "https://ifttt.com/":https://ifttt.com/.
+Finally you'll need an account with Slack and a workspace to send messages to, sign in or create a workspace at "https://slack.com/signin":https://slack.com/signin
+
+h2. Setup
+
+We're going to need an Ably API key for use by your browser application. For our purposes, only one API key is required. If you extend the tutorial with your own functionality (you may wish to create a separate API key for the IFTTT if you need it to have a different set of permissions than those assigned for use by the browser app).
+
+First, log into your Ably dashboard, find the app you're going to be using for this tutorial (the "Sandbox" app is probably fine), then click "Manage App".
+
+<a href="/images/tutorials/reactor/reactor-event-ably-apikey-1.png" target="_blank">
+  <img src="/images/tutorials/reactor/reactor-event-ably-apikey-1.png" style="width: 100%" alt="Manage your Ably app">
+</a>
+
+Select the "*API Keys*" tab, and click "*Create a new API key*":
+
+<a href="/images/tutorials/reactor/reactor-event-ably-apikey-2.png" target="_blank">
+  <img src="/images/tutorials/reactor/reactor-event-ably-apikey-2.png" style="width: 100%" alt="Create a new API key">
+</a>
+
+# Enter a name for your API key
+# Select only the "publish" and "subscribe" options
+# Click the "Create key" button
+
+Scroll down if necessary, and copy and paste your new API key to a text file for later use in the application.
+
+h2. Creating an Ably Reactor Rule
+
+We'll need to create a Reactor Event Rule in your Ably dashboard. This is what will trigger the IFTTT event.
+
+# Go to your Ably Dashboard
+# Choose the "Reactor" tab
+# Click "New Reactor Rule" button
+# Click the "Choose" button under "Reactor Event" type
+
+h2. Creating an IFTTT Reactor Event
+
+Choose the "*IFTTT*" service integration option
+
+On the resulting form we'll need an IFTTT key to connect Ably to IFTTT with webhooks. 
+
+# Click the "open IFTTT Webhook Settings" button (this will take you to the IFTTT web app)
+# On the resulting page click "Documentation"
+# Copy your key from the top of the documentation page
+# Go back to your Ably Reactor Integration Rule Settings
+# Paste the key into the "IFTTT Webhook key" input.
+# Give the Webhook event a name (eg channel_message_published)
+# Set up a channel filter if necessary
+# Click "*Create*"
```

Comment:
A screenshot of the set up, with private keys blurred, would be a nice touch at each step.  Sadly this is all quite dry in that a developer has no visual reference at each stage of the tutorial.